NOnion/Crypto/TorCryptoState.fs

```diff
@@ -19,8 +19,8 @@ type TorCryptoState =
         : TorCryptoState =
         let fCipher = TorStreamCipher(kdfResult.ForwardKey, None)
         let bCipher = TorStreamCipher(kdfResult.BackwardKey, None)
-        let fDigest = TorMessageDigest()
-        let bDigest = TorMessageDigest()
+        let fDigest = TorMessageDigest(kdfResult.IsHSV3)
+        let bDigest = TorMessageDigest(kdfResult.IsHSV3)
```

Comment:
nit: no need for parentheses in the 2 lines above

Reply:
btw, what's the rationale for those weird variable names? fDigest? bDigest? what does f and b mean?

Reply:
Done